plugins/azure/sqldatabases/dbDataDiscoveryClassification.js

```diff
@@ -0,0 +1,70 @@
+var async = require('async');
+var helpers = require('../../../helpers/azure');
+
+module.exports = {
+    title: 'Data Discovery and Classification for SQL Databases',
+    category: 'SQL Databases',
+    domain: 'Databases',
+    description: 'Use data discovery and classification to classify the sensitivity of your data on SQL databases.',
```

Comment:
```suggestion
    description: 'Ensures data discovery and classification is implemented for SQL databases.',
```

should be what the plugin does and not the recommended action